pom.xml

```diff
@@ -15,8 +14,13 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+        <commons.version>1.10</commons.version>
+        <httpcomponents.version>4.5.1</httpcomponents.version>
+        <script-security.version>1.17</script-security.version>
+        <workflow.version>1.13</workflow.version>
```

Comment:
@jglick I do not know how to do what you are asking. I tried this:
```
--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.3</version>
+        <version>1.580.1</version>
     </parent>
 
     <artifactId>http_request</artifactId>
@@ -18,7 +18,7 @@
         <commons.version>1.10</commons.version>
         <httpcomponents.version>4.5.1</httpcomponents.version>
         <script-security.version>1.17</script-security.version>
-        <workflow.version>1.13</workflow.version>
+        <workflow.version>1.4.2</workflow.version>
     </properties>
```

But it gave me:
```
[ERROR] /home/martin/git/jenkinsci/http-request-plugin/src/main/java/jenkins/plugins/http_request/HttpRequestStep.java:[18,43] error: cannot find symbol
[ERROR] symbol:   class AbstractSynchronousNonBlockingStepExecution
[ERROR] location: package org.jenkinsci.plugins.workflow.steps
```